### PR TITLE
bithumb - fix comma number

### LINKED ISSUE #11031: fetchOpenOrders error on bithumb
MUST READ THIS BEFORE SUBMITTING ISSUES (read the links, then delete this message before submitting):

https://github.com/ccxt/ccxt/wiki/FAQ
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue
Make sure your local version of CCXT is up to date. Check by comparing the output of ccxt.version to https://github.com/ccxt/ccxt/blob/master/package.json#L3

OS: win10
Programming Language version:
CCXT version: 1.64.70

I thought the below problem was solved.
But I'm still getting the error.
please check.

I can't speak English, so I'm getting help from Google Translate.
Please understand even if my English is wrong.

There is a problem with OpenOrder lookup in Bithumb.
Problems with certain tickers (VET/KRW)

1,115.0508 is the quantity in OpenOrder.

bithumb.fetchOpenOrders('VET/KRW')

Traceback (most recent call last):

File "", line 1, in
bithumb.fetchOpenOrders('VET/KRW')

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\bithumb.py", line 758, in fetch_open_orders
return self.parse_orders(data, market, since, limit)

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 1942, in parse_orders
array = [self.extend(self.parse_order(order, market), params) for order in orders]

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 1942, in
array = [self.extend(self.parse_order(order, market), params) for order in orders]

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\bithumb.py", line 723, in parse_order
}, market)

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 2511, in safe_order2
amount = self.omit_zero(self.safe_string(order, 'amount'))

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 2661, in omit_zero
if float(string_number) == 0:

ValueError: could not convert string to float: '1,115.0508'

If there is no open order for any ticker
When looking at the ticker, the following error occurs.

bithumb.fetchOpenOrders('ETH/KRW')
Traceback (most recent call last):

File "", line 1, in
bithumb.fetchOpenOrders('ETH/KRW')

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\bithumb.py", line 739, in fetch_open_orders
response = self.privatePostInfoOrders(self.extend(request, params))

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 500, in inner
return entry(_self, **inner_kwargs)

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 549, in request
return self.fetch2(path, api, method, params, headers, body, config, context)

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 545, in fetch2
return self.fetch(request['url'], request['method'], request['headers'], request['body'])

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 697, in fetch
self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\bithumb.py", line 850, in handle_errors
self.throw_exactly_matched_exception(self.exceptions, status, feedback)

File "C:\Users\HwanyL2\Anaconda3\lib\site-packages\ccxt\base\exchange.py", line 563, in throw_exactly_matched_exception
raise exactstring

ExchangeError: bithumb {"status":"5600","message":"\uac70\ub798 \uc9c4\ud589\uc911\uc778 \ub0b4\uc5ed\uc774 \uc874\uc7ac\ud558\uc9c0 \uc54a\uc2b5\ub2c8\ub2e4."}

### PR DESCRIPTION
**Pull request by ttodua**
fix #11031

### CODE REVIEW

File: js/bithumb.js [hunk 1]
```diff
@@ -1031,6 +1009,18 @@ module.exports = class bithumb extends Exchange {
         };
     }
 
+    fixCommaNumber (numberStr) {
+        // some endpoints need this https://github.com/ccxt/ccxt/issues/11031
+        if (numberStr === undefined) {
+            return undefined;
+        }
+        let finalNumberStr = numberStr;
+        while (finalNumberStr.indexOf (',') !== -1) {
```
Comment (carlosmiei) [inline js/bithumb.js:1018]: @ttodua, I'm a bit confused, why do we need this loop?

Isn't the `replace` function to replace all occurrences on the first call, or am I missing something?

```
const x = "122,33,44,55"
const y = x.replace(",", "") // y = 122334455

Comment (frosty00) [reply js/bithumb.js:1018]: actually it doesn't, that would be replaceAll or replace (/regex/g)

Comment (ttodua) [reply js/bithumb.js:1018]: yes, as carlo said.  (we dont have regex codes transpilable, so had to use that approach). alternatively, we might add 'replaceG/replaceAllOccurences' method in base. thoughts?

Comment (carlosmiei) [reply js/bithumb.js]: @ttodua @frosty00  yeah, my bad what I said is only valid for PY and PHP, not JS

image
image

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**